### PR TITLE
feat: searching tools

### PR DESCRIPTION
**Pull request by Kojot999**

### CODE REVIEW

File: src/app/api/api.js [hunk 1]
```diff
@@ -1,21 +1,9 @@
...
-//     .join("");
-
-//   const result = API_BASE + path + query + "&key=" + API_KEY;
-//   return result;
-// };
-
-// const result = queryBuilder("/games", exampleParams);
-// console.log(result);
+  const result = RAWG_API + "/games" + API_KEY + query;
+  console.log(result);
```
Comment (pislagz) [inline src/app/api/api.js:7]: delete

File: src/app/api/games/index.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import { composeUrl } from "../utils";
+import { queryBuilder } from "../api";
 
-const getGamesData = async () =>
-  await (await fetch(composeUrl("/games"))).json();
+const getGamesData = async ({ search }) => {
+  // to pojdzie do queryBuildera
```
Comment (pislagz) [inline src/app/api/games/index.js:4]: delete

**Review by pislagz — CHANGES_REQUESTED** [review]

File: src/app/api/api.js (hunk 1 above)
Comment (Kojot999) [reply src/app/api/api.js:7]: done

File: src/app/api/games/index.js (hunk 2 above)
Comment (Kojot999) [reply src/app/api/games/index.js:4]: done

**Pull request merged**